.changeset/beige-eagles-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/cli-node': patch
+---
+
+Updated doc link.
```

Comment:
Do we need setup redirects from the old urls to the new ones to avoid broken links?

Reply:
We done have short links for some bits, but didn't bother with this one ofc. I'd say it's fine to have this one be broken for a lil bit